static/prelogin/less/components/pricing-section.less

```diff
@@ -16,12 +16,12 @@
 @pricing-fourth-section-height-xs: 486px;
 @pricing-fourth-section-height-sm: 486px;
 @pricing-fourth-section-height-md: 481px;
-@pricing-fourth-section-height-lg: 555px;
+@pricing-fourth-section-height-lg: 496px;
```

Comment:
ya forgot about the other sections https://github.com/dimagi/commcarehq-prelogin/pull/70